homeassistant/components/totalconnect/quality_scale.yaml

```diff
@@ -0,0 +1,65 @@
+rules:
+  # Bronze
+  config-flow: done
```

Comment:
The flows could be supported by some more context using `data_description`. Also, I am missing the fields of the reauth form, so purely looking at the code it makes me believe that we don't show `password` at all there

Reply:
set to `todo`